openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/SuggestionRepository.java

```diff
@@ -220,6 +238,70 @@ protected void acceptSuggestion(
     update(suggestion, user);
   }
 
+  protected void acceptSuggestionList(
+      List<Suggestion> suggestions, SecurityContext securityContext, Authorizer authorizer) {
+    String user = securityContext.getUserPrincipal().getName();
```

Comment:
we need to validate if this user has permissions to edit descripton or edit tags